examples/tutorials/physics_benchmarking.py

```diff
@@ -0,0 +1,334 @@
+    if "graphics_render_times" in data:
+        data["graphics_render_times"] += graphics_render_times
+    if "collisions" in data:
+        data["collisions"] += collisions
+
+
+# Define each test function & store in list
+
+
+def box_drop_test(
+    objects,
+    num_objects=30,
+    box_size=2,
+    object_speed=2,
+    post_throw_settling_time=5,
+):  # take in parameters here
+    """Drops a specified number of objects into a box and returns metrics including the time to simulate each frame, render each frame, and the number of collisions each frame. """
+    # start with box being bounding boxes
+
+    data = {
+        "observations": [],
+        "physics_step_times": [],
+        "graphics_render_times": [],
+        "collisions": [],
+    }
+
+    cfg = make_configuration()
+    with habitat_sim.Simulator(cfg) as sim:
+        sim.initialize_agent(
+            0,
+            habitat_sim.AgentState(
+                [5.45 * box_size, 2.4 * box_size, 1.2 * box_size],
+                np.quaternion(-0.83147, 0.2, -0.55557, -0.2),
+            ),
+        )
+
+        # get the physics object attributes manager
+        obj_templates_mgr = sim.get_object_template_manager()
+        # [/initialize]
+
+        # build box
+        cube_handle = obj_templates_mgr.get_template_handles("cube")[0]
+
+        box_parts = []
+        boxIDs = []
+        for _i in range(5):
+            box_parts += [obj_templates_mgr.get_template_by_handle(cube_handle)]
+        box_parts[0].scale = np.array([1.0, 0.1, 1.0]) * box_size
+        box_parts[1].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[2].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[3].scale = np.array([0.1, 0.6, 1.0]) * box_size
+        box_parts[4].scale = np.array([0.1, 0.6, 1.0]) * box_size
+
+        for i in range(5):
+            part_name = "box_part_" + str(i)
+            obj_templates_mgr.register_template(box_parts[i], part_name)
+            boxIDs += [sim.add_object_by_handle(part_name)]
+            sim.set_object_motion_type(
+                habitat_sim.physics.MotionType.KINEMATIC, boxIDs[i]
+            )
+        sim.set_translation(np.array([2.50, 0.05, 0.4]) * box_size, boxIDs[0])
+        sim.set_translation(np.array([2.50, 0.35, 1.30]) * box_size, boxIDs[1])
+        sim.set_translation(np.array([2.50, 0.35, -0.50]) * box_size, boxIDs[2])
+        sim.set_translation(np.array([3.40, 0.35, 0.4]) * box_size, boxIDs[3])
+        sim.set_translation(np.array([1.60, 0.35, 0.4]) * box_size, boxIDs[4])
+        for i in range(5):
+            sim.set_object_motion_type(habitat_sim.physics.MotionType.STATIC, boxIDs[i])
+
+        # load some object templates from configuration files
+        object_ids = []
+        scales = {
+            "test_assets/objects/sphere": 1,
+            "test_assets/objects/chair": 0.9,
+            "test_assets/objects/donut": 2,
+            "test_assets/objects/nested_box": 0.5,
+        }
```

Comment:
Maybe want to embed this metadata in the object parameter so it can be configured externally? Could be tuples or dictionaries for each entry.